docs/community.rst

```diff
@@ -13,6 +13,7 @@ Communication channels
 * Questions tagged ``kinto`` on `Stack Overflow <http://stackoverflow.com/questions/tagged/kinto>`_.
 * Our IRC channel ``#kinto`` on ``irc.freenode.net`` —
   `Click here to access the web client <https://kiwiirc.com/client/irc.freenode.net/?#kinto>`_
+* Get an invite to our slack channel here: https://slack.kinto-storage.org/
```

Comment:
I would phrase it a bit differently: "if you prefer to use slack, head over to https://slack.kinto-storage.org/"